lib/utils/utils.ts

```diff
@@ -115,6 +115,18 @@ export const derivePairId = (pair: Pair) => {
   return `${pair.baseCurrency}/${pair.quoteCurrency}`;
 };
 
+/**
+ * Convert ticker symbole into pair.
+ * @param { string } pair - example: btc/ltc
+ */
+export const getPairCurrencies = (pair: string): Pair => {
```

Comment:
As I've said in previous comments I don't think we need this function. @erkarl what are your thoughts?

If we do keep it I think it can be more concise to match how it's currently being done:

```javascript
const [baseCurrency, quoteCurrency] = pairId.split('/');
return { baseCurrency, quoteCurrency };
```

Reply:
I think the code would read better without this function.

Reply:
@sangaman I removed it.
 I just don't agree tho we use this same function multiple times throughout the code base. and `getPairCurrencies` is clear to me at least